meeting-notes/2018/2018-11-19--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,221 @@
+    - Add core interface tests for [`cidBase` option PR](https://github.com/ipfs/js-ipfs/pull/1552)
+    - Write interop tests for CIDv0/1 put and get
+
+
+@hugomrdias
+- Done:
+    - fix ctl options and normalize behavior between the client, daemon and in-proc
+        - Fix non disposable daemon init/start and attach to running daemons [#308](https://github.com/ipfs/js-ipfsd-ctl/pull/308)
+    - fix: add missing dependencies [#1663](https://github.com/ipfs/js-ipfs/pull/1663)
+    - Fails to init with Node.js 11.x - Lock FcntlFlock failed: EINVAL, Invalid argument [#1700](https://github.com/ipfs/js-ipfs/issues/1700)
+        - fix: fix lock for node 11 [#181](https://github.com/ipfs/js-ipfs-repo/pull/181)
+    - fix: remove electron-webrtc and wrtc for now [#1718](https://github.com/ipfs/js-ipfs/pull/1718)
+    - [WIP] fix: remove dexie and unused deps [#309](https://github.com/ipfs/js-ipfsd-ctl/pull/309)
+    - chore: add renovate [#1722](https://github.com/ipfs/js-ipfs/pull/1722)
+    - update gitlab ci config and added docs according to the feedback
+    - started working on libp2p-mplex async iterators
+- Blocked
+    - fix: pass config to init [#303](https://github.com/ipfs/js-ipfsd-ctl/pull/303)
+    - [WIP] feat: Try out CircleCI's new infrastructure for test & release flow #1669
+    - CI for js-ipfs [#442](https://github.com/ipfs/infrastructure/issues/442)
+    - Fix non disposable daemon init/start and attach to running daemons [#308](https://github.com/ipfs/js-ipfsd-ctl/pull/308)
+    - chore: add renovate [#1722](https://github.com/ipfs/js-ipfs/pull/1722) need renovate service active if we think we should test it.
+- Next
+    - bubble up [#181](https://github.com/ipfs/js-ipfs-repo/pull/181)
+    - continue working on libp2p-mplex async iterators
+    - continue bundle size work
+    - score okr
+
+@achingbrain
+ - Done:
+  - Split ipfs-unixfs-engine into importer and exporter modules
+    - Removed a few redundant dependencies
+    - Added tests for directory sharding to unixfs-exporter
+    - Refactored unixfs-exporter tests to use in-memory `ipld`
+    - Made the exporter tests run in a browser (!)
+  - PR that removes CIDs from DAGNodes was merged to `js-ipfs` supporting modules
+  - Compat: support `count` as well as `length` arg https://github.com/ipfs/js-ipfs-mfs/pull/27
+  - Allow writing mfs dirs/files with difference CID versions & hashes to the containing directory https://github.com/ipfs/js-ipfs-mfs/pull/26
+  - Use `ipld.get(cid, path) in unixfs-exporter` https://github.com/ipfs/js-ipfs-unixfs-engine/issues/177
+  - Return last traversed CID from `ipld.get` https://github.com/ipld/js-ipld/pull/181
+  - Use `ipld.getMany` in unixfs-exporter https://github.com/ipfs/js-ipfs-unixfs-exporter/pull/2
+  - Implement directory sharding for mfs
+ - Blocked:
+   - Object API changes preventing release of DAGNode CID removal
+   - DAGNode CID removal PR: https://github.com/ipfs/js-ipfs/pull/1702
```

Comment:
FYI these changes have been bundled up into https://github.com/ipfs/js-ipfs/pull/1720 and will be merged today pending passing CI.